Assets/Scripts/Game/MagicAndEffects/EntityEffectManager.cs

```diff
@@ -785,8 +786,14 @@ public void UseItem(DaggerfallUnityItem item)
                         EntityEffectBundle bundle = new EntityEffectBundle(bundleSettings, entityBehaviour);
                         SetReadySpell(bundle, true);
 
-                        // TODO: Apply durability loss to used item on use
+                        // Apply durability loss to used item on use
                         // http://en.uesp.net/wiki/Daggerfall:Magical_Items#Durability_of_Magical_Items
+                        item.currentCondition -= 10;
+                        if (item.currentCondition <= 0 && collection != null)
```

Comment:
Bonus, thank you! :)

Reply:
Figured while I was in the vicinity... :+1: